kibble/render/render.go

```diff
@@ -111,13 +126,21 @@ func Render(sourcePath string, buildPath string, cfg *models.Config) int {
 			Language:    createLanguage(cfg, lang, locale),
 		}
 
-		if lang != cfg.DefaultLanguage {
-			ctx.RoutePrefix = fmt.Sprintf("/%s", lang)
-			i18n.LoadTranslationFile(filepath.Join(sourcePath, ctx.Language.DefinitionFilePath))
-		} else {
-			i18n.MustLoadTranslationFile(filepath.Join(sourcePath, ctx.Language.DefinitionFilePath))
+		if apiTranslations != nil {
+			file, err := json.Marshal(apiTranslations[locale])
+			if err != nil {
+				log.Errorf("Failed to marshal translations json %s: %s", locale, err)
+				return 1
+			}
+			err = ioutil.WriteFile(filepath.Join(sourcePath, ctx.Language.DefinitionFilePath), file, 0644)
```

Comment:
I think it would be good to explain why we are writing this file and what the permissions mean.